LICENSE

```diff
@@ -1,29 +1,28 @@
-Copyright (c) Acoular Development Team.
-All rights reserved.
+BSD 3-Clause License
 
+Copyright (c) 2024, acoular
```

Comment:
To be consistent, I would prefer to use the `Copyright (c) Acoular Development Team` statement without specifying a certain year. 
```suggestion
Copyright (c) Acoular Development Team
```